packages/canvas-tokens/utils/transformers/index.ts

```diff
@@ -28,6 +28,12 @@ export const transforms: Record<string, Transform> = {
     matcher: filter.isSysColor,
     transformer: flatRGBAColor,
   },
+  'value/opacity': {
+    type: 'value',
+    transitive: true,
+    matcher: filter.isBaseOpacity,
+    transformer: ({value}) => `${value / 100}`,
```

Comment:
I always forget that JS is cool with doing math on string numbers.

Reply:
if it's only numbers, no letters lol